[
    {
        "\u00e9quipe": "020",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 77, in <module>\n    format_final(ARGS.symbole)\n  File \"projet1.py\", line 75, in format_final\n    deuxieme_ligne(k[symb])\n  File \"projet1.py\", line 67, in deuxieme_ligne\n    response['Time Series (Daily)'][e][c],\nKeyError: 'Time Series (Daily)'\n</pre>\n"
    },
    {
        "\u00e9quipe": "038",
        "score": 68,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3>"
    },
    {
        "\u00e9quipe": "001",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.</p>\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n"
    },
    {
        "\u00e9quipe": "003",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.</p>\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n"
    },
    {
        "\u00e9quipe": "004",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 78, in <module>\n    tuplefinal(debut, fin, site(i))\n  File \"projet1.py\", line 70, in tuplefinal\n    if str(debut1) in response['Time Series (Daily)'].keys():\nKeyError: 'Time Series (Daily)'\n</pre>\n"
    },
    {
        "\u00e9quipe": "006",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 87, in <module>\n    L[len(L) - 1] = L[len(L) - 1] + ')'\nIndexError: list index out of range\n</pre>\n"
    },
    {
        "\u00e9quipe": "007",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.</p>\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n"
    },
    {
        "\u00e9quipe": "008",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 95, in <module>\n    fonc(liste) ## fin TP 19 novembre\n  File \"projet1.py\", line 47, in fonc\n    v_debut = response['Time Series (Daily)'][d[0]]\nKeyError: 'Time Series (Daily)'\n</pre>\n"
    },
    {
        "\u00e9quipe": "009",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 91, in <module>\n    fonction(ARGS.symbole, ARGS.debut, ARGS.fin, ARGS.valeur)\n  File \"projet1.py\", line 72, in fonction\n    if str(une_date) in response['Time Series (Daily)']:\nKeyError: 'Time Series (Daily)'\n</pre>\n"
    },
    {
        "\u00e9quipe": "011",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.</p>\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n"
    },
    {
        "\u00e9quipe": "012",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 89, in <module>\n    out()\n  File \"projet1.py\", line 65, in out\n    if str(DATE1 + timedelta(i)) in response['Time Series (Daily)']:\nKeyError: 'Time Series (Daily)'\n</pre>\n"
    },
    {
        "\u00e9quipe": "013",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.</p>\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n"
    },
    {
        "\u00e9quipe": "014",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 104, in <module>\n    affichage(ARGS.symbole, ARGS.debut, ARGS.fin, ARGS.valeur)\n  File \"projet1.py\", line 80, in affichage\n    if str(delai) in response['Time Series (Daily)']:\nKeyError: 'Time Series (Daily)'\n</pre>\n"
    },
    {
        "\u00e9quipe": "015",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 99, in <module>\n    valeur1 = dureepluspetite()['Time Series (Daily)']['{}'.format(dated)][clef]\nKeyError: 'Time Series (Daily)'\n</pre>\n"
    },
    {
        "\u00e9quipe": "016",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.</p>\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n"
    },
    {
        "\u00e9quipe": "017",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 180, in <module>\n    main()\n  File \"projet1.py\", line 167, in main\n    response = response['Time Series (Daily)']\nKeyError: 'Time Series (Daily)'\n</pre>\n"
    },
    {
        "\u00e9quipe": "018",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"Projet1.py\", line 75, in <module>\n    fonction_premiere()\n  File \"Projet1.py\", line 68, in fonction_premiere\n    response['Time Series (Daily)']\nKeyError: 'Time Series (Daily)'\n</pre>\n"
    },
    {
        "\u00e9quipe": "019",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 90, in <module>\n    print(sortie(ARGUMENTS.v, response, DAT))\n  File \"projet1.py\", line 41, in sortie\n    respons['Time Series (Daily)'][dats[0].strftime('%Y-%m-%d')][ty]))\nKeyError: 'Time Series (Daily)'\n</pre>\n"
    },
    {
        "\u00e9quipe": "021",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.</p>\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n"
    },
    {
        "\u00e9quipe": "022",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 73, in <module>\n    for i in RESPONSE['Time Series (Daily)']:\nKeyError: 'Time Series (Daily)'\n</pre>\n"
    },
    {
        "\u00e9quipe": "023",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.</p>\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n"
    },
    {
        "\u00e9quipe": "024",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> S.V.P. limitez l'utilisation du programme \u00e0 5 requ\u00eates par minute.\n</pre>\n"
    },
    {
        "\u00e9quipe": "025",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Il n'y a pas de fichier projet1.py dans le dossier de votre bundle.</p>"
    },
    {
        "\u00e9quipe": "026",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.</p>\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n"
    },
    {
        "\u00e9quipe": "027",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 90, in <module>\n    newdico = dicoresponse['Time Series (Daily)']\nKeyError: 'Time Series (Daily)'\n</pre>\n"
    },
    {
        "\u00e9quipe": "029",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 82, in <module>\n    fct1()\n  File \"projet1.py\", line 75, in fct1\n    if response['Time Series (Daily)'].get(str(date1)) is not None:\nKeyError: 'Time Series (Daily)'\n</pre>\n"
    },
    {
        "\u00e9quipe": "029",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Il n'y a pas de fichier projet1.py dans le dossier de votre bundle.</p>"
    },
    {
        "\u00e9quipe": "030",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Il n'y a pas de fichier projet1.py dans le dossier de votre bundle.</p>"
    },
    {
        "\u00e9quipe": "030",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 226, in <module>\n    affichage(s)\n  File \"projet1.py\", line 200, in affichage\n    dic_bourse = response['Time Series (Daily)'][date_ouver_converti]\nKeyError: 'Time Series (Daily)'\n</pre>\n"
    },
    {
        "\u00e9quipe": "031",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 22, in <module>\n    sortie()\n  File \"projet1.py\", line 9, in sortie\n    duree = setup.jours_semaine()\n  File \"/mnt/Documents/Projets/Laval/TA/Partie1/unbundled/bundle-031-02.hg/setup.py\", line 93, in jours_semaine\n    info = information('goog')\n  File \"/mnt/Documents/Projets/Laval/TA/Partie1/unbundled/bundle-031-02.hg/setup.py\", line 65, in information\n    return response['Time Series (Daily)']\nKeyError: 'Time Series (Daily)'\n</pre>\n"
    },
    {
        "\u00e9quipe": "032",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 97, in <module>\n    new_dict = response['Time Series (Daily)']\nKeyError: 'Time Series (Daily)'\n</pre>\n"
    },
    {
        "\u00e9quipe": "033",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 124, in <module>\n    print('{}({}, {}, {})'.format(infos.indice_boursiers(), infos.variable,\n  File \"projet1.py\", line 97, in indice_boursiers\n    return self.dictio['Meta Data']['2. Symbol']\nKeyError: 'Meta Data'\n</pre>\n"
    },
    {
        "\u00e9quipe": "035",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet1.py\", line 6, in <module>\n    import numpy\nModuleNotFoundError: No module named 'numpy'\n</pre>\n"
    },
    {
        "\u00e9quipe": "036",
        "score": 0,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3><p>Votre programme n'a pas fonctionn\u00e9 durant la correction...</p>\n<p>Un premier test (<code>[]</code>) a \u00e9t\u00e9 effectu\u00e9 mais votre programme \u00e0 soulev\u00e9 une erreur.\nIl y a peut-\u00eatre une boucle infinie ou une erreur d'impl\u00e9mentation sur le fonctionnement basique.</p>\n<p>Votre erreur : <p>\n<pre> Traceback (most recent call last):\n  File \"projet-1.py\", line 6, in <module>\n    import holidays\nModuleNotFoundError: No module named 'holidays'\n</pre>\n"
    },
    {
        "\u00e9quipe": "039",
        "score": 73,
        "commentaires": "<h3>R\u00e9sultat crit\u00e8re 3</h3>\n<p>Vous avez \u00e9chou\u00e9 le test : <CODE>--valeur=volume --fin=2018-09-24 --d\u00e9but=2018-09-21 goog\n</CODE></p>\n<p>La bonne r\u00e9ponse est : </p>\n<pre>goog(volume, 2018-09-21, 2018-09-24)\n[('2018-09-21', '4405584'), ('2018-09-24', '1271017')]\n</pre>\n<p>Votre r\u00e9ponse est : </p>\n<pre>[]\nMais elle n'a pas le format attendu\n</pre><p>Votre erreur est : </p>\n<pre> usage: projet1.py [-h] [-d DATE] [-f DATE]\n                  [-v {fermeture,ouverture,min,max,volume}]\n                  symbole [symbole ...]\nprojet1.py: error: unrecognized arguments: --d\u00e9but=2018-09-21\n</pre>"
    }
]
